### PR TITLE
Merging readmes and pipeline

### PR DESCRIPTION
**Pull request by SurinaCS**
Adding pipeline fully cloud-based to main. The readmes are all updated to reflect current state of repo. AWS architecture is planned with terraform and implemented.
Addressing #129

### CODE REVIEW

File: clean/README.md
Comment (Kurt812) [inline clean/README.md:1]: Nice clean up of unused code

File: dashboard/README.md
Comment (Kurt812) [inline dashboard/README.md:1]: kept dashboard readme up to date

File: README.md [hunk 1]
```diff
@@ -103,18 +100,25 @@ Each directory has its own `.env` or secrets file. Please refer to the individua
...
 This directory contains all code relating to the dashboard that users can use to track and submit topics to monitor their trends.
 
 - **[images](https://github.com/Kurt812/trend-getters-project/tree/main/images)**
 This directory contains all the images found in this repository including the Entity Relationship Diagram (ERD) and the architecture diagram.
+
+- **[notifications](https://github.com/Kurt812/trend-getters-project/tree/main/notifications)**
+This directory contains all the code relating to collecting subscription data and sending notifications.
+
+- **[notifications-pipeline](https://github.com/Kurt812/trend-getters-project/tree/main/notifications-pipeline)**
+This directory contains all the code relating to updating the data for the keywords that are subscribed to before sending the notifications.
```
Comment (keoghrmj) [inline README.md:114]: great explanations!

File: images/logo-no-writing.png
Comment (Kurt812) [inline images/logo-no-writing.png:1]: Nice organisation with images

**Comment by Kurt812** [comment]
All files were checked when merged to development, code looking good.

File: .gitignore
Comment (Kurt812) [inline .gitignore:1]: Secrets protected successfully

**Review by Kurt812 — APPROVED** [review]
Nice pull to main, no changes needed.

**Pull request merged**